flake.nix

```diff
@@ -0,0 +1,448 @@
+{
+  description = "BDK Flake to run all tests locally and in CI";
+
+  inputs = {
+    # stable nixpkgs
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-23.05";
+    # pin bitcoind to a specific version
+    # find instructions here:
+    # <https://lazamar.co.uk/nix-versions>
+    # pinned to 0.25.0
+    nixpkgs-bitcoind.url = "github:nixos/nixpkgs?rev=9957cd48326fe8dbd52fdc50dd2502307f188b0d";
+    # Blockstream's esplora
+    # inspired by fedimint CI
+    nixpkgs-kitman.url = "github:jkitman/nixpkgs/add-esplora-pkg";
+    crane = {
+      url = "github:ipetkov/crane";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
+    rust-overlay = {
```

Comment:
**thought:** I tend to use `fenix`, and I vaguely remember it working better than "overlays"... but maybe these were "mozilla overlays" back then? Anyway. Whatever works, but just worth pointing out that `fenix` exists.

Reply:
Yes, but I am used to `oxalica` and I have seem it being used more than `fenix`